### DISCUSSION TITLE
Alarm Control Panel

### DISCUSSION THREAD

**Discussion by PM04290** [dawidchyrzynski/arduino-home-assistant#117] [Ideas]
is it planed to add "Alarm Control Panel" device has is possible in MQTT discovery ?

**Comment by mattster98** [comment]
That's what I'm here for.  Might try to hack around it somehow, but I have a ScramblePad hacked in as my garage door opener code entry keypad and am migrating from Homeseer to HA.

I'm guessing it can be worked around by using buttons and some additional automations.  We'll see.
